docs/build/program/apis/data-client.md

```diff
@@ -1009,7 +1009,7 @@ Rename a dataset specified by the dataset ID.
 
 **Parameters:**
 
-- `id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The ID of the dataset.
+- `id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The ID of the dataset. To retrieve this, navigate to your dataset's page in the [Viam app](https://app.viam.com/data/datasets) and copy the string after `?id=` in the page's URL.
```

Comment:
talked to data team/design and they're going to add UI element for this in meeting hopefully tomorrow, will update!